14/umbraco-cms/reference/management-api/README.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Get started with the Management API.
+---
+
+# Management API
+
+The Management API delivers headless capabilities built directly into Umbraco. Unlike the Content Delivery API, the Management API enables interaction with everything else in the backoffice except for the Content and Media items.
+
+The API returns responses in a JSON format, which can be presented in different channels using your preferred technology stack. This feature maintains the user-friendly editing experience of Umbraco, while also ensuring an efficient delivery of content in a headless manner. Additionally, with its different extension points, the API can be customized to meet a wide range of requirements.
+
+The Management API can also be used for Custom apps or Workflows with OpenID Connect.
+
+{% hint style="info" %}
+The Management API is a replacement for the backoffice controllers that lacked RESTful capabilities.
+{% endhint %}
+
+## Swagger Documentation
+
+Umbraco ships with Swagger to document the Management API. Swagger and the Swagger UI are based on [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/) and is available at `{yourdomain}/umbraco/swagger`. For security reasons, both are disabled in non-production environments.
+Read more about Swagger in the [API versioning and OpenAPI](../api-versioning-and-openapi.md) article.
```

Comment:
Be mindful that the linked article solely concerns itself with the Delivery API. While the techniques in the article could theoretically be applied to the Management API, it would be a really bad idea to do so. On top of that, there is a big risk of confusing the two here, which is definitively bad - the Delivery API is inherently public, while the Management API certainly is not.

Reply:
Ah makese sense then! I will remove the sentence linking to API versioning to not cause these confusions 👍